index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header id="header" >
+        <span id="logo">Abdullah Asghar</span>
+        <button type="button" class="btn">
+            <img src="./icon/Icon-Menu.png" alt="Menu-icon">
+        </button>
```

Comment:
- _(optional)_ Please as mentioned in the requirements, kindly wrap your hamburger menu icon with a `nav` tag.